ld: error: cannot open WRKSRC/libmdns/../libmdns: Is a directory
ld: error: unable to find library -lmdns
cc: error: linker command failed with exit code 1 (use -v to see invocation)

--- libmdns/Makefile.orig	2018-05-27 08:03:30 UTC
+++ libmdns/Makefile
@@ -7,7 +7,6 @@ CFLAGS+= -Wmissing-declarations
 CFLAGS+= -Wshadow -Wpointer-arith -Wcast-qual
 CFLAGS+= -Wsign-compare
 CFLAGS+= -I${.CURDIR} -I${.CURDIR}/../mdnsd
-LDADD+= ${.CURDIR}/../libmdns -lmdns
 LDADD+= -lutil
 DPADD+=  ${LIBUTIL}
 
